### DISCUSSION TITLE
Understanding tensorboard graphs

### DISCUSSION THREAD

**Discussion by JRMeyer** [coqui-ai/TTS#51] [Archived]
<USER>>>> haqkiemdaim<br/> <DATE> [May 27, 2019, 4:39pm]<br/> <br/> Hi! I got a few question regarding the graphs visualized on the<br/> Tensorboard:<br/> <br/> 1. Why there is **no Validation Loss graph** but instead EvalStat<br/> graph *slash* <br/> visualized there? Since during training i provide both Training set<br/> and *slash* <br/> Validation set.<br/> <br/> 2. Why there is Evaluation Stat graph there? since i never provide<br/> Testing set *slash* <br/> during the training?<br/> <br/> What is the purpose of having **EvalStat graph** and what can we<br/> know *slash* <br/> from that graph?<br/> <br/> 3. Among **loss_decoder**, **loss_postnet** and **stop_loss**, which<br/> one *slash* <br/> should be really important and to be focused on ?<br/> <br/> 4. How can i know that my current models is overfitting or not since<br/> no *slash* <br/> Validation graph provided??<br/> <br/> [This is an archived TTS discussion thread from discourse.mozilla.org/t/understanding-tensorboard-graphs]<br/>

**Comment by JRMeyer** [comment]
<USER>>>> erogol<br/> <DATE> [May 28, 2019, 12:01am]<br/> <br/> Don't 'overfit' to names. If you mentally replace the name 'Validation'<br/> with 'Evaluation', you solve your problems. In short, check Eval stats<br/> for validation.<br/> <br/> If you are looking for the final voice performance of the model,<br/> 'loss_postnet' is the target. But for the best, they all should<br/> converge.<br/> <br/> [Archived Post]<br/>

**Comment by JRMeyer** [comment]
<USER>>>> haqkiemdaim<br/> <DATE> [June 1, 2019, 8:17am]<br/> <br/> Anwyay, im quite confuse, why do we need both **Eval and Train Audios<br/> generated** such as on the Tensorboard?<br/> <br/> Sory to ask but, can you explain about this?<br/> <br/> About **TestAudios**, is it generated by picking up phonemes from random<br/> sentences/audios?<br/> <br/> [Archived Post]<br/>

**Comment by JRMeyer** [comment]
<USER>>>> erogol<br/> <DATE> [June 3, 2019, 2:04pm]<br/> <br/> Eval == Validation, Train == Training<br/> <br/> If you check train.py you see test audios are generated by a certain<br/> list of sentences.<br/> <br/> [Archived Post]<br/>
